telegram/ext/dispatcher.py

```diff
@@ -275,25 +279,22 @@ def run_async(self, func, *args, update=None, error_handling=True, **kwargs):
         Warning:
             * If you're using ``@run_async``/:meth:`run_async` you cannot rely on adding custom
               attributes to :class:`telegram.ext.CallbackContext`. See its docs for more info.
-            * When using :meth:`run_async` within an error handler (see :meth:`add_error_handler`),
-              you *must* also set :attr:`error_handling` to :obj:`False`, as an exception within
-              :attr:`func` would cause an infinite loop otherwise. Eventual exceptions will then
-              only be logged.
+            * :meth:`run_async` must not be used within an error handler, as an exception within
+              :attr:`func` would cause an infinite loop otherwise.
```

Comment:
I'd try to make it more clear, maybe something like:
```suggestion
            * Calling a function wrapped with `run_async` from within an error handler can lead to an infinite error handling loop.
```